flake.nix

```diff
@@ -460,6 +460,9 @@
                 nativeBuildInputs = shellCommonNative.nativeBuildInputs ++ [ pkgs.yarn pkgs.nodejs ];
                 shellHook = ''
                   export FEDIMINT_UI_SHELL=1
+                  cd fedimint-ui
+                  yarn install
+                  yarn build
```

Comment:
Why is yarn build needed?

Reply:
`yarn build` is needed because we have a monorepo with shared packages.
under `fedimint-ui/apps` we have web applications that use `@fedimint/ui`, which is defined in `fedimint-ui/packages/ui`. By invoking `yarn build` from `fedimint-ui` it compiles everything in `fedimint-ui/packages` so they can be referenced in `fedimint-ui/apps`

If you do not, you will get import errors when attempting to import from `@fedimint-ui`.